book/src/generalized-ownership.md

```diff
@@ -0,0 +1,279 @@
+# Generalized Ownership
+
+This design describes generalized ownership of countable items on the Solana
+blockchain. The goal of this design is to propose a single general-purpose
+program whose state can be used across many different programs to implement
+ownership, transfer and programmatic assignment. 
```

Comment:
programmatic?

Reply:
Assignment based on the output of a program?  What should you call it?